cmd/es-rollover/app/actions_test.go

```diff
@@ -74,21 +73,20 @@ func TestExecuteAction(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			v := viper.New()
-			tlsFlags := tlscfg.ClientFlagsConfig{Prefix: "es"}
 			command := cobra.Command{}
 			flags := &flag.FlagSet{}
-			tlsFlags.AddFlags(flags)
+			AddTLSFlags(flags)
+			AddFlags(flags)
 			command.PersistentFlags().AddGoFlagSet(flags)
 			v.BindPFlags(command.PersistentFlags())
```

Comment:
you should be able to replace this with a single call to Viperize from pkg/config, e.g.
```
v, command := config.Viperize(AddTLSFlags , AddFlags)
```